### ISSUE TITLE
Term Request: Descemet membrane endothelial keratoplasty graft detachment

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11764]
**Preferred term label:**
Descemet membrane endothelial keratoplasty graft detachment

**Synonyms:**
* DMEK graft detachment
* DMEK graft dislocation
* Endothelial keratoplasty graft detachment
* Detachment of DMEK lenticule

**Definition (free text):**
Partial or complete separation and lack of adherence of the transplanted donor Descemet membrane and endothelium from the recipient posterior stroma following Descemet membrane endothelial keratoplasty (DMEK). This condition commonly leads to localized or diffuse persistent corneal stromal edema and may require rebubbling or regrafting.

**Parent term (use hpo.jax.org/app):**
HP:6001448

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
